sql/tsql/TSqlParser.g4

```diff
@@ -1922,6 +1924,23 @@ create_index
     ';'?
     ;
 
+// https://docs.microsoft.com/en-us/sql/t-sql/statements/create-columnstore-index-transact-sql?view=sql-server-ver15
+create_columnstore_index
+    : CREATE (CLUSTERED | (NONCLUSTERED)? ) COLUMNSTORE INDEX id_ ON table_name  
+    index_options?  
+    (ON id_)? 
+    ';'?
+    ;
+  
+// https://docs.microsoft.com/en-us/sql/t-sql/statements/create-columnstore-index-transact-sql?view=sql-server-ver15
+create_nonclustered_columnstore_index
+    : CREATE NONCLUSTERED? COLUMNSTORE INDEX id_ ON table_name '(' column_name_list_with_order ')'
+    (WHERE where=search_condition)?
```

Comment:
Do you really need the label `where`? It can be accessed via just calling `context.search_condition()`. I suggest not using labels without an urgent need (for instance, ambiguity).

```suggestion
    (WHERE search_condition)?
```